Jenkinsfile

```diff
@@ -0,0 +1,44 @@
+pipeline {
+	agent any
+	options {
+		buildDiscarder(logRotator(numToKeepStr:'10'))
+	}
+	stages {
+		stage('Prepare') {
+			steps {
+				git url: 'https://github.com/eclipse/wildwebdeveloper.git'
+				cleanWs()
+				checkout scm
+			}
+		}
+		stage('Build') {
+			steps {
+				wrap([$class: 'Xvnc', useXauthority: true]) {
```

Comment:
Temporarily, let's remove/comment this wrapper until we have Xvnc installed on the machine or can run it in a container.